install_files/securedrop-ossec-agent/var/ossec/etc/ossec.conf

```diff
@@ -5,7 +5,7 @@
 
   <syscheck>
     <alert_new_files>yes</alert_new_files>
-
+    <frequency>25200</frequency>
```

Comment:
These changes look good - but why every 7 hours?

Reply:
[Syscheck frequency default is 6h](https://ossec-docs.readthedocs.io/en/latest/manual/syscheck/) and incorrectly assumed that daily update/reboot would have interrupted the scans. However, it appears that production deployments of ossec do syschecks every ~30 mins. Syscheck frequency is therefore likely not the cause of #2495 . Will close this PR and scope one just for #2496.